jobs/ci-kubernetes-e2e-gke-ubuntustable1-k8sstable1-alphafeatures.env

```diff
@@ -3,11 +3,9 @@
 # The common configurations.
 
 # The cloud provider configurations.
-KUBERNETES_PROVIDER=gke
 E2E_MIN_STARTUP_PODS=8
 ZONE=us-central1-f
```

Comment:
Remove this here and for the other gke-ubuntustable1 jobs? `bazel run //experiment:generate_tests -- --yaml-config-path=experiment/test_config.yaml --json-config-path=jobs/config.json --prow-config-path=prow/config.yaml && bazel run //jobs:config_sort` should work.